compiler/backend_inkwell/candy_runtime/candy_runtime.h

```diff
@@ -25,12 +25,19 @@ typedef struct
     struct candy_value **values;
 } candy_struct_t;
 
+typedef struct
+{
+    char *text;
```

Comment:
In other parts of the compiler, we called this `symbol`, but `text` works as well